### DISCUSSION TITLE
Install from git / pre release version?

### DISCUSSION THREAD

**Discussion by hpoul** [wagtail/wagtail-localize#422] [Q&A] [answered]
Hello,
I wanted to try out wagtail_localize with wagtail 2.13.1 and saw that it was only recently added support. So I thought i'd just `pipenv install` it from the git url. Unfortunately it seems the `static/` folder is generated during release.
Is there some (easy) way to install an pre-release/nightly version? Are there any nightly builds, or similar? (I guess I could download it, run `npm install` and afterwards `pip install .` (?) but it would be nice if I could add it to my `Pipfile`)

Thanks,
  Herbert

**Comment by kaedroho** [comment, marked as answer]
Hi @hpoul,

Apologies for the delay in supporting 2.13, I'll try to sync up Wagtail Localize releases with Wagtail core releases in future so you don't have to wait as long.

2.13 support was released in Wagtail Localize 1.0RC2, which you can upgrade to now!

Thanks,
Karl

**Comment by sheralim012** [comment]
Hi @kaedroho,
As mentioned in the question the `static/` folder is generated during release (empty directory in github codebase but PyPI/github releases have files in it)

I know `npm run build` populates `static/` directory with assets but how can I install from the git URL and still get `static/` directory with assets?
